Extra_tasks/Extra_task_3/Modal/style.css

```diff
@@ -0,0 +1,104 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+}
+
+:root {
+    --margin-l: 1.1rem;
+    --padding-s: 0.5rem;
+    --gap-l: 2.1rem;
```

Comment:
use structure:
--padding-xs: calc(var(--padding) / 4);
--padding-s: calc(var(--padding) / 2);
--padding: 1rem;
--padding-d: calc(var(--padding) * 1.25);
--padding-m: calc(var(--padding) * 1.5);
--padding-l: calc(var(--padding) * 2);
--padding-xl: calc(var(--padding) * 4);
  For marging ang gap same